.gitignore

```diff
@@ -20,12 +20,10 @@
 /static
 /tmp
 celeryd.pid
```

Comment:
No longer relying on hsctl to dynamically generate docker-compose. SOLR schema.xml file needs to be tracked in git.